test/backend_comparison.in.cpp

```diff
@@ -164,6 +185,27 @@ NGRAPH_TEST(${BACKEND_NAME}, lesseq)
     EXPECT_EQ((vector<char>{1, 0, 1, 0, 1, 1, 0, 1}), read_vector<char>(result));
 }
 
+NGRAPH_TEST(${BACKEND_NAME}, lesseq_int32)
```

Comment:
This `lesseq` test only tests less than, doesn't cover the equality case.

Reply:
Good point..

Reply:
However, the equality condition will alway pass for something like `0x40000005` since they will fall back to `0x40000000`. But it makes sense to add this anyway.